src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -727,7 +727,8 @@ private static void validatePropertySection(String fileName, String sectionName,
 
         assertWithMessage(
                 fileName + " section '" + sectionName + "' should show properties: " + properties)
-                        .that(properties).isEmpty();
+                        .that(properties)
```

Comment:
I think that the wrapping on the line #1422 is more readable.
Such indent looks like it is bound to the string message.

Reply:
Done.